packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -394,7 +406,15 @@ module.exports = {
       fileName: 'asset-manifest.json',
       publicPath: publicPath,
     }),
-  ],
+    // TypeScript type checking
+    paths.isTypeScript &&
+      new ForkTsCheckerWebpackPlugin({
+        async: false,
+        tsconfig: paths.appTSConfig,
+        tslint: paths.useTSLint ? paths.appTSLint : undefined,
+        watch: paths.appSrc,
+      }),
```

Comment:
Just wondering why wont we use `formatter: 'codeframe'` option, so we can get typescript errors with big and nice code frames like syntax (which is checked by babel) errors does.

Reply:
@borisowsky thanks, just added this, looks like